### PR TITLE
SCRUM-9: Get_Activities

### PR DESCRIPTION
**Pull request by NicoleMunozT**

### CODE REVIEW

File: tests/Calendar/test_GetActivities.py
Comment (Jasonn5) [inline tests/Calendar/test_GetActivities.py:1]: approved

File: src/resources/schemas/activity.json
Comment (Jasonn5) [inline src/resources/schemas/activity.json:1]: approved

File: src/espocrm_api/activities_endpoints.py
Comment (Jasonn5) [inline src/espocrm_api/activities_endpoints.py:1]: approve

File: src/assertions/assertion_status.py
Comment (Jasonn5) [inline src/assertions/assertion_status.py:1]: it's the same you didn't change anything

File: src/assertions/assertion_schemas.py
Comment (Jasonn5) [inline src/assertions/assertion_schemas.py:1]: approved

File: src/assertions/assertion_content.py
Comment (Jasonn5) [inline src/assertions/assertion_content.py:1]: approved

File: src/assertions/assertion_calendar.py
Comment (Jasonn5) [inline src/assertions/assertion_calendar.py:1]: approved

File: config.py
Comment (Jasonn5) [inline config.py:1]: approved

**Review by Jasonn5 — APPROVED** [review]

File: config.py
Comment (AlisonGuzman2006) [reply config.py:1]: This global variable in the config.py is aproved

File: src/assertions/assertion_calendar.py
Comment (AlisonGuzman2006) [reply src/assertions/assertion_calendar.py:1]: The assert methods are also aproved, thay have clean code.

File: src/assertions/assertion_status.py
Comment (AlisonGuzman2006) [reply src/assertions/assertion_status.py:1]: I agree, its the same, there is no change

File: src/espocrm_api/activities_endpoints.py
Comment (AlisonGuzman2006) [reply src/espocrm_api/activities_endpoints.py:1]: The clas enum created is aproved

File: src/resources/schemas/activity.json
Comment (AlisonGuzman2006) [reply src/resources/schemas/activity.json:1]: The schema created for the Calender endpoint is aproved

File: src/assertions/assertion_content.py
Comment (AlisonGuzman2006) [reply src/assertions/assertion_content.py:1]: These  assert methods are also aproved, thay have clean code.

File: src/assertions/assertion_schemas.py
Comment (AlisonGuzman2006) [reply src/assertions/assertion_schemas.py:1]: The assert methods are also aproved, thay have clean code.

File: tests/Calendar/test_GetActivities.py
Comment (AlisonGuzman2006) [reply tests/Calendar/test_GetActivities.py:1]: The test cases working correctly, and have clean code

**Review by AlisonGuzman2006 — APPROVED** [review]
This pull request has been tested and aproved

**Pull request merged**